Sources/GRPC/AsyncAwaitSupport/GRPCAsyncServerHandler.swift

```diff
@@ -798,7 +803,7 @@ protocol AsyncServerCallContextProvider {
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 @usableFromInline
-internal struct ServerHandlerComponents<Request, Delegate: AsyncWriterDelegate> {
+internal struct ServerHandlerComponents<Request: Sendable, Delegate: AsyncWriterDelegate> {
```

Comment:
I don't think, we should enforce this.

Reply:
We need this one to enforce it for the `PassthroughMessageSource`